Applications/SlicerApp/Testing/Python/JRC2013Vis.py

```diff
@@ -476,19 +453,15 @@ def test_Part4Lung(self):
     #
     # first, get some data
     #
-    import urllib
-    downloads = (
-        ('http://slicer.kitware.com/midas3/download?items=8612', 'LungData.mrb', slicer.util.loadScene),
+    from SampleData import SampleDataSource, SampleDataLogic
+    dataSources = [
+      SampleDataSource(
+        fileNames='LungData.mrb',
+        uris='http://slicer.kitware.com/midas3/download?items=8612',
+        loadFileType='SceneFile'
```

Comment:
I think we already have a mechanism to guess based on file extension. Could we use that by default and require specifying loadFileType only if the default is not good?